Cloud Enterprise/Getting Started Examples/aws/terraform/servers.tf

```diff
@@ -1,78 +1,66 @@
-data "aws_ami" "ubuntu" {
+data "aws_ami" "my-ami" {
   most_recent = true
 
   filter {
     name   = "name"
-    values = ["ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-*"]
+    values = [var.aws_ami_name]
   }
 
   filter {
     name   = "virtualization-type"
     values = ["hvm"]
   }
 
-  owners = ["099720109477"] # Canonical
+  owners = [var.aws_ami_owner]
 }
 
 resource "aws_key_pair" "server" {
-  key_name   = "${var.name}"
-  public_key = "${file(var.public_key)}"
+  public_key = file(var.public_key)
 }
 
 resource "aws_instance" "server" {
-  count = "${length(var.zones)}"
+  count = length(var.zones)
 
-  ami           = "${data.aws_ami.ubuntu.id}"
-  instance_type = "${var.instance_type}"
-  subnet_id     = "${element(aws_subnet.public.*.id, count.index)}"
+  ami           = data.aws_ami.my-ami.id
+  instance_type = var.aws_instance_type
+  subnet_id     = element(aws_subnet.public.*.id, count.index)
 
   vpc_security_group_ids = [
-    "${aws_security_group.administration.id}",
-    "${aws_security_group.servers.id}",
-    "${aws_security_group.internal.id}",
+    aws_security_group.administration.id,
+    aws_security_group.servers.id,
+    aws_security_group.internal.id,
   ]
 
-  key_name = "${aws_key_pair.server.key_name}"
+  key_name = aws_key_pair.server.key_name
 
-  root_block_device {
```

Comment:
Was this one dropped since i3 instances don't support it?

Reply:
Yes, though there is an `ephemeral_block_device` [setting](https://www.terraform.io/docs/providers/aws/r/instance.html#block-devices you can apply, but after a little playing, I didn't get it working right. Some googling showed that it wasn't necessary to configure at all, at least with instances that have only a single drive.